### ISSUE TITLE
admin: show recipient when it can be known

### ISSUE THREAD

**Issue by peter-** [filesender/filesender#900]
When looking through the transfers in the admin UI I notice many having "Anonymous" (with "Direct link provided" displayed on mouse-over) shown in the "Recipient" column but when looking at the transfer it becomes apparent it is from a guest upload (sender email from a domain that cannot log in to FileSender) *but* we have set this in our guest_options[1]:

```php
  'can_only_send_to_me' => array(
    'available' => false,
    'advanced' => false,
    'default' => true
  ),
```

I.e., in our case (or more generally, whenever `can_only_send_to_me` is true for a given transfer) the recipient of a guest upload can't be anonymous, it can only ever be the sender of the guest voucher / upload invitation.

In these cases the hint shown "Direct link provided" is factually incorrect, I think, because `get_a_link` is mutually exclusive with `can_only_send_to_me`, AFAIU, and therefore `get_a_link` cannot have been used since we only allow vouchers with `can_only_send_to_me`.

Would it be possible to show the transfer's recipient, then? Probably the email address of the creator of the transfer (since with this configuration it's impossibile for it to be someone else)?

[1] We do this in order to ensure that at least one leg (i.e., either the sender or the recipient) of any transfer is a member of our community: If person A creates a voucher with `can_only_send_to_me set` (mandatory in our setup) then person A, a member of our community, is the only possible recipient of the transfer (as the author of the guest voucher) even though the sender of the transfer may be any external subject.  
Without that setting person A could create a voucher for person B who could then use this to send files to person C (who may be 2 steps removed from being a community member, even further than person B was as sender of the transfer / recipent of the voucher).

**Comment by monkeyiq** [comment]
I think the setup you describe in [1] where guests can only upload to the person who invited them to the system is quite a useful configuration.  Yes, if an_only_send_to_me is true then get_a_link should be false. I have added some user side control and markings to this effect on the guests invite page (if these are available=true) letting the user know that these options are not able to be set both true. 

As I have these UI elements available to me my intitial testing for this did not show the presentation issue. I then made my config.php set can_only_send_to_me as you mention and saw the presentation bug in the transfers list. I have merged 920 to fix this for new guests.
